array-methods/map.ts

```diff
@@ -0,0 +1,13 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+const objsArr = prices.map((v, i, arr) => {
+  const obj = { price: arr[i], salePrice: arr[i] / 2 /* v */ };
+  return obj;
+});
+console.log('Objects created from prices array:', objsArr);
+const dollarSignFormat = prices.map((v) => {
+  const cents = v * 100;
+  const roundedToNearestCent = Math.round(cents);
+  const strVersion = String(roundedToNearestCent);
+  return `$${strVersion.slice(0, -2)}.${strVersion.slice(-2)}`;
+});
```

Comment:
I'm assuming you had wanted to figure out how to do this yourself, but in case you did not see we have a link to the `toFixed` method in the exercise that would make this easier. It will cut to the number of decimals that you want and if needed round up that last decimal based on the following number.